doc/api/player_events.md

```diff
@@ -560,6 +561,41 @@ You can then know if any of those Representations are becoming decipherable or
 not through their `decipherable` property.
 
 
+<a name="events-inbandEvents"></a>
+### inbandEvents ###############################################################
+
+_payload type_: ``Object``
+
+---
+
+:warning: This event is not sent in _DirectFile_ mode (see [loadVideo
+options](./loadVideo_options.md#prop-transport)).
+
+---
+
+Event triggered when the player encounters inband events in the stream. These
+events are included in the loaded and parsed chunks, and are often used to carry
+content metadata.
+
+Each event contains :
+  - type (_type_: ``String``) : defines the type of the event, specific to an
+  inband event from a streaming protocol.
+  - value (_type_: ``Object``) : the actual parsed content of the event.
+
+The supported inband event types are :
+- "emsg" : The emsg (Event message box) provides inband signaling for generic
+  or MPEG-DASH specific events.
+One ISOBMFF media segment may contain one or several boxes. The parsed event contains :
```

Comment:
It's not clear enough to me that this part is linked to the `"emsg"`  inband event type. Maybe we should align it so it's also aligned in the resulting HTML documentation.